src/custom.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "*.svg" {
+  const content: any;
+  export default content;
+}
```

Comment:
what's the purpose of this file?

Reply:
It's to allow importing svg's in ts files